qa/tasks/mgr/dashboard/helper.py

```diff
@@ -4,14 +4,18 @@
 
 import json
 import logging
+import re
 import time
 from collections import namedtuple
 
 import requests
+
```

Comment:
You need to remove this line, otherwise `isort` will complain about it.
```suggestion
```

```
lint run-test: commands[3] | isort ../../../../qa/tasks/mgr/dashboard --check
ERROR: /ceph/qa/tasks/mgr/dashboard/helper.py Imports are incorrectly sorted and/or formatted.
```

I think it could help if you rebuild your tox environment using `tox -r` to get `tox -e fix` running.
```
╭─root@ceph-master-docker /ceph/src/pybind/mgr/dashboard ‹testing› 
╰─# tox -e fix                                                                                                                
fix installed: astroid==2.4.2,autopep8==1.5.4,docutils==0.16,entrypoints==0.3,flake8==3.7.8,flake8-colors==0.1.6,isort==5.5.3,lazy-object-proxy==1.4.3,mccabe==0.6.1,pycodestyle==2.5.0,pyfakefs==4.1.0,pyflakes==2.1.1,pylint==2.6.0,rstcheck==3.3.1,six==1.15.0,toml==0.10.1,wrapt==1.12.1
fix run-test-pre: PYTHONHASHSEED='1967164192'
fix run-test: commands[0] | python --version
Python 3.8.5
fix run-test: commands[1] | autopep8 --max-line-length 100 --exclude '.tox,
.git,
__pycache__,
build,
dist,
*.egg-info,
.cache,
*.pyc,
.eggs,
venv,
frontend,' --in-place --recursive .
fix run-test: commands[2] | isort .
Skipped 3 files
fix run-test: commands[3] | isort ../../../../qa/tasks/mgr/dashboard
Fixing /ceph/qa/tasks/mgr/dashboard/helper.py
____________________________________________________________ summary _____________________________________________________________
  fix: commands succeeded
  congratulations :)

```

Reply:
No worries. Thanks, will do that!